index.js

```diff
@@ -75,27 +75,24 @@ function gulpRequireTasks (options) {
      */
     function taskNameFromPath (modulePath) {
       const relativePath = path.relative(options.path, modulePath);
-      return removeExtension(relativePath)
-        .split(path.sep).join(options.separator)
-      ;
+      const pathInfos = path.parse(relativePath);
+      var taskNameParts = []
```

Comment:
I can't declare `taskNameParts` as a constant because I change its value later. If you want, I can use the `let` declaration but you don't use it in your plugin.